src/Components/test/testassets/TestServer/Pages/PrerenderedHost.cshtml

```diff
@@ -8,7 +8,7 @@
     <base href="~/" />
 </head>
 <body>
-    <app><component type="typeof(TestRouter)" render-mode="ServerPrerendered" /></app>
+    <app><component type="typeof(TestRouterWithLazyAssembly)" render-mode="ServerPrerendered" /></app>
```

Comment:
```suggestion
    <app><component type="typeof(TestRouterWithLazyAssembly)" render-mode="Static" /></app>
```

This better represents a prerendered webassembly app.

Reply:
Got it. I wasn't sure if changing this would affect any of the other pre-rendering tests so I was conservative about the change but I suppose the existing tests pass then we are good.

Reply:
Update: a few of the existing tests don't pass if the render mode is different.

I did verify that the dynamic assembly works in both static and server prerendered scenarios.

I'll lump this in with the work item to figure out (and test) PWA. We might want to ad a way to test different render modes...